helm/alfresco-content-services/templates/ingress-repository.yaml

```diff
@@ -10,11 +10,13 @@ metadata:
     nginx.ingress.kubernetes.io/session-cookie-hash: "sha1"
     # Default file limit (1m) check, document(s) above this size will throw 413 (Request Entity Too Large) error
     nginx.ingress.kubernetes.io/proxy-body-size: {{ .Values.repository.ingress.maxUploadSize }}
+    {{- if index .Values "alfresco-search" "enabled" }}
```

Comment:
If we disable solr we still could use an external one right? So for this case we would need the redirect rules or? It is probably more save to always forbid access to those solr urls with alfresco as a proxy.